.kokoro/release/stage.cfg

```diff
@@ -5,7 +5,7 @@ timeout_mins: 300
 
 env_vars: {
   key: "TRAMPOLINE_BUILD_FILE"
-  value: "github/google-cloud-java/.kokoro/release/stage.sh"
+  value: "github/google-cloud-java/.kokoro/release/stage_google_oss_sonatype_org.sh"
```

Comment:
@ddixit14 Can you review this change? Is this looking good to you?

Reply:
Yes, I think it makes more sense than having something like `stage_maps.sh` because we might get more artifacts which have the same host.